src/ServerPrivate.cc

```diff
@@ -301,6 +301,17 @@ void ServerPrivate::AddRecordPlugin(const ServerConfig &_config)
               cPathElem->Set<std::string>(cmpPath);
             }
 
+            // If record topics specified, add in SDF
```

Comment:
No action needed on this PR, but just for future reference, we're trying to pass this kind of information to systems through components rather than the SDF now.